### PR TITLE
BLOG: q3 2022

### PR DESCRIPTION
**Pull request by choldgraf**
This is a blog post to provide our external stakeholders an update of what we've been up to in Q3. It tries to synthesize the main points that were shared in https://github.com/2i2c-org/team-compass/issues/515 and the associated HackMD.

### Notes about what is here

- I looked through the HackMD and tried to pick our 3-4 of the most important things for each category. I also lumped things together under `organization wide`, and `service improvements`. I don't currently have a section about `community and partnerships` (see below).
- I didn't include many of the features that @damianavila shared because for many of them, we didn't seem to have any user-facing documentation. Given that these blog posts are meant for end-users and external stakeholders, I think we should be linking to the user docs, not a pull request or issue, so I tended to not include them. We should improve our practices around not considering an issue "done" until it is documented. Please let me know if you disagree with this approach.

### Questions

- There isn't really anything in here specifically about Community or Partnerships improvements. If @colliand or @jmunroe would like to provide information about training material, new partnerships processes and systems, etc, please say so. In the HackMD I only saw items about CSCCE and the strategic lead. Maybe I missed something?
- Am I missing anything important?

### References

- part of https://github.com/2i2c-org/team-compass/issues/515

### CODE REVIEW

**Review by sgibson91 — APPROVED** [review]

File: content/blog/2022/q3-update/index.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+**We are hiring an engineer**: Do you know an open source cloud engineer with experience in Kubernetes and has desire make research and education more impactful, accessible, and delightful? Check out our job ad [at ].
+-->
+
+## Service improvements
+
+We made a number of improvements to our cloud infrastructure and the processes around our service.
+Here's a brief breakdown:
+
+**We expanded our shared clusters to new cloud providers and regions**. We now have a shared clusters already deployed on Google Cloud Platform on `us-central1-b` and `europe-west2` and we will soon deploy a shared cluster on Amazon Web Services on `us-west-2`.
```
Comment (damianavila) [inline content/blog/2022/q3-update/index.md]: I would avoid reporting about things we have not deployed yet, ie. AWS shared cluster.

File: content/blog/2022/q3-update/index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+**We defined an incident commander process**. This will allow us to coordinate and respond to major outages in our cloud infrastructure more efficiently. See [our incident response documentation](https://team-compass.2i2c.org/en/latest/projects/managed-hubs/incidents.html) for more information.
+
+**We improved our cloud usage monitoring infrastructure**. We've deployed [a centralized Grafana Dashboard](https://github.com/2i2c-org/infrastructure/issues/328) that aggregates cloud usage across all of our partner communities, and allows us to keep track of any unexpected behavior or outages across them all. We're also looking into setting up automatic budgets and alerts for our infrastructure.
```
Comment (damianavila) [inline content/blog/2022/q3-update/index.md]: Again, not sure about adding pieces we are currently working on (or planning to be working on).

**Comment by choldgraf** [comment]
@damianavila both of your comments sound reasonable to me - but can you please just suggest edits to the document for the things you want to remove? I don't have the context for what exactly we accomplished this quarter, that is why I rely on you, @jmunroe, and @colliand to make those decisions here. I'm just including whatever folks added to the original hackmd. Please remove (or add) as you wish.

I'd like to publish this on Monday, so please do so before then.

File: content/blog/2022/q3-update/index.md (hunk 1 above)
Comment (damianavila) [inline content/blog/2022/q3-update/index.md]:
```suggestion
**We expanded our shared clusters to new cloud providers and regions**. We now have shared clusters already deployed on Google Cloud Platform on `us-central1-b` and `europe-west2`.
```

File: content/blog/2022/q3-update/index.md (hunk 2 above)
Comment (damianavila) [inline content/blog/2022/q3-update/index.md]:
```suggestion
**We improved our cloud usage monitoring infrastructure**. We've deployed [a centralized Grafana Dashboard](https://github.com/2i2c-org/infrastructure/issues/328) that aggregates cloud usage across all of our partner communities, and allows us to keep track of any unexpected behavior or outages across them all.
```

**Comment by choldgraf** [comment]
I've addressed @damianavila's comments and am going to merge this one in so that we get it out there. if anybody would like to suggest edits or additions post-hoc, please let me know!

**Pull request merged**